index.js

```diff
@@ -88,6 +88,33 @@ const useEslintRc = () => config => {
   return config;
 };
 
+
+const useBabelRc = plugin => config => {
+  const babelLoaderFilter = rule =>
```

Comment:
We may want to move this logic into a reusable utility so that we can share it across different rewirers